components/bl-dropzone-component/src/helpers.js

```diff
@@ -0,0 +1,97 @@
+import { useState } from 'react';
+
+const UploadStatus = {
+  success  : 'success',
+  uploading: 'uploading',
+  undefined: 'undefined',
```

Comment:
what does it mean?

Reply:
'undefined' is a default file upload status. After adding a file to the drop zone, before we click "upload", the file has upload status of "undefined".
Also, if we set the file upload status to "undefined" instead of "uploading",  the file upload will stop.